src/Microsoft.Sbom.Common/Constants.cs

```diff
@@ -26,4 +26,5 @@ public static class Constants
     public const string SPDXRefFile = "SPDXRef-File";
     public const string SPDXRefPackage = "SPDXRef-Package";
     public const string SPDXRefExternalDocument = "DocumentRef";
+    public const string NoAssertionValue = "NOASSERTION";
```

Comment:
This constant is defined in both SPDX2 and SPDX3. That suggests that we're handling things inconsistently. I'm not entirely sure of the best fix. Making the parsers depend on another package doesn't seem great, but neither does having 3 copies of the same data.